lib/foods.js

```diff
@@ -1,28 +1,56 @@
+const local = 'http://localhost:3000/api/foods'
```

Comment:
@TheKaterTot let's be on the watch for this, and maybe audit our JS once every evening to check for this? I agree.